src/Utilities/Header/header.jsx

```diff
@@ -1,24 +1,79 @@
-import React from 'react'
-import './header.css'
-import {Link} from 'react-router-dom';
+import React, { useContext, useEffect } from "react";
+import "./header.css";
+import { Link } from "react-router-dom";
+import { GlobalContext } from "../../Context/GlobalState";
 
 export default function Header() {
+  const { wishlist,cartlist } = useContext(GlobalContext);
   return (
     <>
-        <div className="header">
-                <div className="header-nav-items">
-                  {/* <div className="nav-item"><Link to={"/Games"}>Games</Link></div> */}
-                    <div className="nav-item"><Link to="/Products">Products</Link></div>
-                    {/* <div className="nav-item"><Link to="/Merch">Merch</Link></div> */}
-                    <div className="nav-item"><a href="">Support</a></div>
-                </div>
-                <div className="header-title"> <Link style={{all:"unset"}} to="/"><span>悪い Gamers</span> </Link></div>
-                <div className="header-icons">
-                    <div className="icon-items"><Link to="/Login"> Login/Signup </Link></div>
-                    <div className="icon-items"><Link to="/Wishlist"><i className="far fa-heart"></i></Link></div>
-                    <div className="icon-items"><Link to="/Cart"><i className="fas fa-shopping-cart"></i></Link></div>
-                </div>
+      <div className="header">
+        <div className="header-nav-items">
+          <div className="nav-item">
+            <Link to="/Products">Products</Link>
+          </div>
+          <div className="nav-item">
+            <Link to="/">Support</Link>
+          </div>
         </div>
+        <div className="header-title">
+          {" "}
+          <Link style={{ all: "unset" }} to="/">
```

Comment:
Try to avoid inline styling

Reply:
had to make some quick changes but will do it later